packages/discovery-provider/plugins/notifications/src/email/notifications/preRendered/requestManager.ts

```diff
@@ -0,0 +1,735 @@
+export const email = ({
+  managerName,
+  managerHandle,
+  managerProfilePicture,
+  managedAccountName,
+  managedAccountHandle,
+  managedAccountProfilePicture,
+  link
+}: {
+  managerName: string
+  managerHandle: string
+  managerProfilePicture: string
+  managedAccountName: string
+  managedAccountHandle: string
+  managedAccountProfilePicture: string
+  link: string
+}) => {
+  return `
+  <!doctype html>
```

Comment:
Just gonna go ahead and trust this even though what is going on with all the tables 😅